lib/containers/runtime.pm

```diff
@@ -108,6 +160,24 @@ sub enum_containers {
     return \@containers;
 }
 
+=head2 get_images_by_repo_name
+
+Returns an array ref with the names of the images.
+
+=cut
+sub get_images_by_repo_name {
+    my ($self) = @_;
+    my $repo_images;
+    unless ($self->runtime eq 'buildah') {
+        $repo_images = $self->_rt_script_output("images --format '{{.Repository}}'");
+    }
+    else {
+        $repo_images = $self->_rt_script_output("images --format '{{.Name}}'");
+    }
+    my @images = split /[\n\t ]/, $repo_images;
```

Comment:
What is this for?

Reply:
what do you mean? 
get an array from the string